client/src/components/organisms/ImageGallery/ImageGallery.js

```diff
@@ -1,6 +1,11 @@
 import * as React from 'react';
 import { ImagesWrapper, MiniImagesWrapper } from './ImageGallery.styles';
 
+const setImageSrc = (data) =>
+  data && data.url
+    ? data.url.replace('http://', 'https://')
+    : '/product-placeholder.jpeg';
```

Comment:
I see that this `/product-placeholder.jpeg` was initially only used in the main image, should we maybe include this back?

Reply:
wow it's late, just realised what I said :joy: approving!